blog/authors.yml

```
@@ -1,10 +1,4 @@
-  url: https://github.com/KorayErkan
-  image_url: https://avatars.githubusercontent.com/u/129009941?s=400&u=f85c95cb082c30fde4aa695f6d1e3c84e21e73ac&v=4
-  title: Technical Writer
-  page:
-    permalink: '/all-vm-articles'
-  socials:
-    linkedin: KorayErkan
-    github: KorayErkan
+team:
+  name: VirtualMetric Team
+  title: Release Engineering
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)